spec/brcobranca/boleto/caixa_spec.rb

```diff
@@ -117,10 +117,8 @@
     expect(boleto_novo.agencia_conta_boleto).to eql('2030/654321-9')
   end
 
-  it 'Busca logotipo do banco' do
-    boleto_novo = described_class.new
-    expect(File.exist?(boleto_novo.logotipo)).to be_truthy
-    expect(File.stat(boleto_novo.logotipo).zero?).to be_falsey
+  describe 'Busca logotipo do banco' do
+    it_behaves_like 'busca_logotipo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.